src/styles/_cards.scss

```diff
@@ -117,3 +117,9 @@
     }
   }
 }
+
+@media (max-width: 1023px) {
```

Comment:
If you add auto-fill and minmax in your .catalog you won't need a media query. 
Like so `grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));`